src/pages/legal/tos.jsx

```diff
@@ -0,0 +1,110 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash">
+        <div id="masthead">
+          <div className="inner">
+            <div id="header">
+              <a href="#/"><img src="./img/logo.png" width="200" height="53" alt="Webmaker" /></a>
+            </div>
+          </div>
+        </div>
+
+        <div id="mid">
+          <div className="inner legal">
+            <h1>Webmaker & teach.mozilla.org Terms of Use</h1>
+
+            <h3>June 15, 2013</h3>
+
+            <p>At Mozilla, we are committed to promoting choice and innovation on the web and helping users realize the full potential of the web. We created Webmaker and [teach.mozilla.org] (collectively, “Sites”) to improve web literacy, by enabling users to learn about web technologies by creating apps and submitting them for others to mash up. and providing resources for a community of educators and activists who want to teach digital skills through making.”</p>
```

Comment:
That's pretty tedious to add. Turning wrapping on in an editor is easy enough...